presto-native-execution/presto_cpp/main/SessionProperties.cpp

```diff
@@ -0,0 +1,242 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "presto_cpp/main/SessionProperties.h"
+#include "velox/core/QueryConfig.h"
+
+using namespace facebook::presto;
```

Comment:
Its a bit strange to have using namespace facebook::presto and then making it the namespace of this class as well. Is this using statement needed ?

Reply:
Removed it, thanks for pointing it out.